markdown/developers/graphic_design.md

```diff
@@ -0,0 +1,58 @@
+---
+
+nf-core has a variety of recognisable images and other graphic designs for the community project. To help facilitate the identification of nf-core pipelines and related material, we provide here a variety of files that can be used for all nf-core-related content, e.g. pipeline logos, presentations, or workflow diagrams.
+
+All files are released under various forms of 'open' licenses (see each item), meaning you can re-use and re-model (sometimes with attribution) as you see fit.
+
+## Logo
+
+All information for the nf-core logo can be seen at the [nf-core/logos GitHub repository](https://github.com/nf-core/logos).
+
+## Fonts
+
+The font(s) typically used in nf-core graphic design (particularly logos) is [Maven Pro](https://fonts.google.com/specimen/Maven+Pro), which is released under a Open Font License (see link for details).
+
+The exact variants of the fonts can also be seen at the [nf-core/logos GitHub repository](https://github.com/nf-core/logos).
+
+## Colours
+
+The nf-core green colour is as follows:
+
+* RGB: 36,176,100
+* Hex: #24B064.
+
+## Presentation Templates
+
+nf-core regularly host [community events](https://nf-co.re/events), such as hackathons, and encourage training the wider bioinformatics community in creating nextflow and nf-core pipelines.s
+
+If you wish to introduce or run training sessions to your group or network, you can access a Google Doc template **here**.
+
+## Workflow Schematics
+
+### Examples
+
+It is often useful to have simplified diagrams that outline the main functionality and steps of a pipeline. We include here examples of different types from other pipelines that can be re-used and modified for your own.
+
+Original files can be seen on the respective pipeline GitHub repositories, which can be accessed via the corresponding links.
+
+#### Workflow: nf-core/sarek
+
+<p align="center">
+    <img src="https://raw.githubusercontent.com/nf-core/sarek/master/docs/images/sarek_workflow.png" alt="nf-core sarek workflow" width="50%">
+</p>
+
+Workflow diagram from [nf-core/sarek](https://github.com/nf-core/sarek/) and published in [Garcia _et al._ (2020, F1000 Research)](https://doi.org/10.12688/f1000research.16665.1) under a [CC-BY 4.0](https://creativecommons.org/licenses/by/4.0/) license.
+
+### Components
+
+Furthermore, there are some sub-components (such as icons) of the above example workflows made by members of the community, that are generally applicable to every pipeline. A couple of examples of these can be seen below, but are all accessible on the [nf-co.re website GitHub repository](https://github.com/nf-core/nf-co.re/assets/graphic_design/workflow_schematics/subcomponents), and can be re-used for your own graphics.
+
+<p align="center">
+    <img src="../../assets/graphic_design/workflow_schematics_components/sarek/single_fastq.png" alt="Single FASTQ Icon">
+    <img src="../../assets/graphic_design/workflow_schematics_components/sarek/multiple_vcf.png" alt="Multiple VCF Icon">
+</p>
```

Comment:
Maybe we can do this as a table instead, with links to the SVG for example..

Reply:
Will get back to this later this week - but good idea! Do you mean as a markdown table?

Reply:
Yup 👍 There are a bunch of online markdown table generators that make this doable.